Source/WebCore/Modules/compression/CompressionStreamEncoder.h

```diff
@@ -27,6 +27,9 @@
 #include "BufferSource.h"
 #include "ExceptionOr.h"
 #include "Formats.h"
+#if PLATFORM(COCOA)
```

Comment:
Yeah, having a separate implementation of the same thing lends itself to having a platform specific file.

Reply:
I agree.

Currently prototyping a new version that adds new files for the zlib and compression streams backends.